dart/lib/src/http_client/sentry_http_client.dart

```diff
@@ -83,7 +83,6 @@ class SentryHttpClient extends BaseClient {
     MaxRequestBodySize maxRequestBodySize = MaxRequestBodySize.never,
```

Comment:
Similar to https://github.com/getsentry/sentry-dart/pull/816/files#r840296083

Can also be moved to SentryOptions